Workbooks/WorkbooksMetadata.json

```diff
@@ -2204,5 +2204,18 @@
                 "Platform"
             ]
         }
-    }      
+    },   
+    {
+      "workbookKey": "MicrosoftSentinelCostEUR",
+      "logoFileName": "Azure_Sentinel.svg",
+      "description": "This workbook provides an estimated cost in EUR (€) across the main billed items in Microsoft Sentinel: ingestion, retention and automation. It also provides insight about the possible impact of the Microsoft 365 E5 offer.",
+      "dataTypesDependencies": [],
+      "dataConnectorsDependencies": [],
+      "previewImagesFileNames": [ "MicrosoftSentinelCostEURWhite.png", "MicrosoftSentinelCostEURBlack.png"],
+      "version": "1.6.0",
+      "title": "Microsoft Sentinel Cost (EUR)",
```

Comment:
@MarcoPassanisi, Title should not have () can you please check and give title something as "Microsoft Sentinel Cost-EUR", thanks.

Reply:
I chose this name because I saw that there is a name for the GBP currency and I have aligned with it.